ors-engine/src/main/java/org/heigit/ors/isochrones/GraphEdgeMapFinder.java

```diff
@@ -114,6 +118,6 @@ private static AccessibilityMap calculateTimeDependentAccessibilityMap(Isochrone
 
     private static Weighting createWeighting(IsochroneSearchParameters parameters, FlagEncoder encoder) {
         return parameters.getRangeType() == TravelRangeType.TIME ? new FastestWeighting(encoder)
-                : new DistanceWeighting(encoder);
+                : new ShortestWeighting(encoder);
     }
```

Comment:
DistanceWeighting differs from ShortestWeighting wrt. details of how the edge weight and minimum edge weight are computed. Are we sure that replacing DistanceWeighting has no undesired side-effects?